run.py

```diff
@@ -3,20 +3,25 @@
 import os
 
 
-path_to_runner = "./Debug/SCP.exe"
+path_to_runner = "./x64/Debug/SCP.exe"
```

Comment:
This may not be the same for all machines depending on the run configuration. Perhaps have both paths and check which one is present on the host machine